src/components/login/Login.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Copyright from './Copyright';
+import useStyles from './style';
+import LogoImage from '../aa.png';
+import getUserLogin from '../API/getUserLogin';
+const Login = (props: any) => {
+  const classes = useStyles();
+  const [user, setLoginUser] = useState([]);
```

Comment:
Why do you have array of users here? Also please add a type for the user. 

useState<User>